tfhe/src/integer/ciphertext/mod.rs

```diff
@@ -1,490 +1,11 @@
-//! This module implements the ciphertext structures.
+mod base;
```

Comment:
let's keep things public, as you can see the boolean_value one was public, same thing for shortint please

Reply:
Done